components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -299,57 +334,90 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 
 	if isA2HARBFileExist() {
 
-		var err error
 		infra, err := getAutomateHAInfraDetails()
 		if err != nil {
 			return err
 		}
+		timestamp := time.Now().Format("20060102150405")
 
-		sshUser := infra.Outputs.SSHUser.Value
-		sskKeyFile := infra.Outputs.SSHKeyFile.Value
-		sshPort := infra.Outputs.SSHPort.Value
+		configFile := args[0]
+		frontend := &Cmd{
+			PreExec: prePatchCheckForFrontendNodes,
+			CmdInputs: &CmdInputs{
+				Cmd:        fmt.Sprintf(FRONTEND_COMMAND, PATCH, "frontend"+"_"+timestamp+"_"+configFile, dateFormat),
+				Single:     false,
+				Args:       args,
+				InputFiles: []string{configFile},
+				ErrorCheck: true,
+				NodeType:   configCmdFlags.frontend,
+			},
+		}
 
-		timestamp := time.Now().Format("20060102150405")
-		sshConfig := &SSHConfig{
-			sshUser:    sshUser,
-			sshKeyFile: sskKeyFile,
-			sshPort:    sshPort,
+		automate := &Cmd{
+			PreExec: prePatchCheckForFrontendNodes,
+			CmdInputs: &CmdInputs{
+				Cmd:        fmt.Sprintf(FRONTEND_COMMAND, PATCH, "automate"+"_"+timestamp+"_"+configFile, dateFormat),
+				Single:     false,
+				Args:       args,
+				InputFiles: []string{configFile},
+				ErrorCheck: true,
+				NodeType:   configCmdFlags.automate,
+			},
+		}
+
+		chef_server := &Cmd{
```

Comment:
Rename this `chef_server` to camel case.

Reply:
Done